app/src/main/java/io/apicurio/registry/rest/v2/AdminResourceImpl.java

```diff
@@ -307,7 +307,8 @@ public void close() throws IOException {
     @Audited(extractParameters = {"0", KEY_FOR_BROWSER})
     @Authorized(style=AuthorizedStyle.None, level=AuthorizedLevel.Admin)
     public Response exportData(Boolean forBrowser) {
-        if (forBrowser != null && forBrowser) {
+        String acceptHeader = request.getHeader("Accept");
+        if ((forBrowser != null && forBrowser) || (acceptHeader != null && MediaType.APPLICATION_JSON.equals(acceptHeader))) {
```

Comment:
Looks good, but `acceptHeader != null` check is not needed, because the next part covers null.